spark.md

```diff
@@ -31,4 +31,24 @@ SciSpark was installed using the [instructions available at SciSpark GitHub page
 
 A python notebook to do [Unsupervised classification of imagery using scikit-learn](http://nbviewer.jupyter.org/gist/om-henners/c6c8d40389dab75cf535). This example shows how to classify imagery (for example from LANDSAT) using scikit-learn. There are many classification methods available, but for this example they use K-Means as it's simple and fast. It uses [**numpy**](http://www.numpy.org/) and [**rasterio**](https://github.com/mapbox/rasterio).
 
+## Debug mode
+All information to debug set Spark for remote debugging and performance tuning.
 
+### Remote debugging
+To set Spark for remote debugging the user should reconfigure Spark to have only an executor per worker. In **emma/vars/spark_vars/.yml** the user should set *spark_executor_cores* equal to *spark_worker_cores* and *spark_executor_memory* equal to *spark_worker_memory*. Such setting allows us to open a single debugger per executor, i.e., one per node.
```

Comment:
There should be something about `spark_debug_mode` here.  If the mode==False than stuff below does not work. Please describe how to enable Spark debug mode.

Reply:
I forgot to push that, it is now in.
Good catch